tests/UnitBLAS/Level2.hs

```diff
@@ -66,6 +66,26 @@ matmatTest1ZGEMV = do
     resList <- Matrix.mutableVectorToList $ _bufferMutDenseVector res 
     resList @?= [2.0,2.0]
 
+----
+----
+
+matmatTest1SGER :: IO ()
+matmatTest1SGER = do
+  res <- Matrix.generateMutableDenseMatrix (Matrix.SRow) (2,2) (\_ -> 1.0)
+  x <- Matrix.generateMutableDenseVector 2 (\_ -> 2.0)
+  y <- Matrix.generateMutableDenseVector 2 (\_ -> 3.0)
+  BLAS.sger 2.0 x y res
+  resList <- Matrix.mutableVectorToList $ _bufferDenMutMat res
```

Comment:
might be good to add a wee comment explaining 
A := alpha*x*y' + A, with the number filled in. So that *why* this test is correct is more immediately obvious.

Which does raise the realted point that we should probably put those formulae in the haddcoks somewhhere, but probably not in the tests or this PR